### PR TITLE
add CHANGES file

### PR DESCRIPTION
**Pull request by raffenet**
## Pull Request Description

Highlight additions and changes since the 0.1x branch was made.

## Author Checklist
* [x] Reference appropriate issues (with "Fixes" or "See" as appropriate)
* [x] Commits are self-contained and do not do two things at once
* [x] Commit message is of the form: `module: short description` and follows [good practice](https://chris.beams.io/posts/git-commit/)
* [x] Add comments such that someone without knowledge of the code could understand
* [x] Have read and agree to the Yaksa CLA terms (https://github.com/pmodels/yaksa/wiki/Yaksa-Contributor-License-Agreement)

### CODE REVIEW

**Comment by raffenet** [comment]
I tried to capture as many significant changes as I could from the beginning of 0.1x to 0.2x.
```
git shortlog --no-merges 536645c33e21fb2770d8071c0e9b329095915e5a..0.2x
```

File: CHANGES [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+===============================================================================
+
+# Add support for reduction operations (e.g. sum, prod, min, max, ...)
+
+# Add support for AMD GPUs via HIP backend
+
+# Add support for NVIDIA HPC SDK compilers
+
+# Add blocking pack/unpack APIs
+
```
Comment (hzhou) [inline CHANGES:12]: We also added `yaksa_pack/unpack_stream` -- https://github.com/pmodels/yaksa/commit/ef90cf608bd3f9c412e3b804653e00ef4c9d29a5

Comment (hzhou) [reply CHANGES:12]: Oh, maybe that is already in `0.1x`?

Comment (hzhou) [reply CHANGES:12]: https://github.com/pmodels/yaksa/pull/169 -- Add `nogpu` info hint -- is also worth to mention.

Comment (raffenet) [reply CHANGES:12]: No, I see it in the shortlog. Added a line for it.

**Review by hzhou — APPROVED** [review]
LGTM!

File: CHANGES (hunk 1 above)
Comment (raffenet) [reply CHANGES:12]: > #169 -- Add `nogpu` info hint -- is also worth to mention.

[we discussed offline] This patch was actually included in the 0.1x branch used for MPICH 3.4.3 which is why I originally left it out, but I do think it is worth a mention on second thought. Will add.

**Pull request merged**